app/src/main/java/io/appium/uiautomator2/model/settings/MjpegScalingFactor.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.uiautomator2.model.settings;
+
+import io.appium.uiautomator2.server.ServerConfig;
+
+public class MjpegScalingFactor extends AbstractSetting<Integer> {
+    private static final String SETTING_NAME = "mjpegScalingFactor";
+
+    public MjpegScalingFactor() {
+        super(Integer.class, SETTING_NAME);
+    }
+
+    @Override
+    public Integer getValue() {
+        return ServerConfig.getMjpegScalingFactor();
+    }
+
+    @Override
+    protected void apply(Integer value) {
+        ServerConfig.setMjpegScalingFactor(value);
```

Comment:
Do we want to apply a validation for setting values?

Reply:
It seems like most of the other `Setting(s)` **do not** validate. I think the doc-string comment is a reasonable starting point. Longer term it may make sense to add a `validate` method to the `AbstractSetting` class.

Reply:
I'm ok with that, but it usually makes sense to at least check if the given value is in the acceptable range and reset it to default/throw exception if not in case these values are received from the client (people sometimes don't know what they are doing and not reading the documentation ;) )

Reply:
Fair enough -- This will be resolved w/ the upcoming update(s)